README.md

```diff
@@ -1,3 +1,42 @@
-# IDR metadata
+# IDR studies
 
 [![Published screens](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fscreens%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20high%20content%20screens)](https://idr.openmicroscopy.org) [![Published experiments](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fprojects%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20experiments)](https://idr.openmicroscopy.org)
+
+All metadata associated with published studies in IDR is managed
+
+## Study name 
+
+After acceptance, IDR studies must be named as `idr<NNNN>-<name>-<description>`
+where `idr<NNNN>` is the accession number of the study using an incremental
+four digits integer, `<author>` is the name of one of the authors associated
```

Comment:
four-digit

Reply:
`<author>` should be `<name>` (or the other way round earlier).